src/App.svelte

```diff
@@ -0,0 +1,115 @@
+<script>
+	let display = 0;
+	let firstOperand = null;
+	let secondOperand = false;
+	let operator = null;
+	let operatorClicked = false;
+
+	const addDisplay = (num) => {
+		if(display === 0 || operatorClicked ) {
+			display = '';	
+		}
+		if(display.length <= 13){
+			display += String(num);	
+		}
+		operatorClicked = false;
+	}
+
+	const dot = () => {
+		if(display == 0) {
+			display = '0.';
+		}else if(!display.includes('.')){
+			display += '.';
+		}	
+	}
+
+	const resetDisplay = () => {
+		display = 0;
+	}
+
+	const deleteNum = () => {
+		display = String(display);
+		if(display.length <= 1) { 
+			display = 0;
+		}else if (display == /.$/) {
+			display = display.slice(0, -2); 
+		}else{
+			display = display.slice(0, -1);
+		}	
+	}	
+	
+	const equal = () => {
+		//debugger;
```

Comment:
You should remove this if you don't need anymore.

Reply:
Removed